storage/files.rb

```diff
@@ -386,6 +386,46 @@ def generate_signed_url project_id:, bucket_name:, file_name:
   # [END generate_signed_url]
 end
 
+def generate_signed_get_url_v4 project_id:, bucket_name:, file_name:
+  # [START storage_generate_signed_url_v4]
+  # project_id  = "Your Google Cloud project ID"
+  # bucket_name = "Your Google Cloud Storage bucket name"
+  # file_name   = "Name of a file in the Cloud Storage bucket"
```

Comment:
nit: You use "Google Cloud Storage" and then "Cloud Storage".
Be consistent either way.